corgidrp/l1_to_l2a.py

```diff
@@ -154,7 +154,7 @@ def prescan_biassub(input_dataset, noise_maps=None, return_full_frame=False,
 
     return output_dataset
 
-def detect_cosmic_rays(input_dataset, detector_params, k_gain, sat_thresh=0.7,
+def detect_cosmic_rays(input_dataset, k_gain = None, detector_params = None, sat_thresh=0.7,
```

Comment:
sorry, I was a bit unclear here: I meant that kgain should be the first optional keyword (i.e., `input_dataset, detector_params, kgain=None, sat_thresh=0.7, ...`)

Reply:
Understood, changed back again. I think now it should be ready for review.